recipes/exiv2/all/conanfile.py

```diff
@@ -44,8 +44,6 @@ class Exiv2Conan(ConanFile):
         "win_unicode": False,
     }
 
-    provides = []
```

Comment:
is there any difference between having "provides = None" and not using it explicitly?

Reply:
Yes, in conan client code, when checking dependencies graph, is expected `None` is case providing nothing, using list it will result in an undesirable behavior:

https://github.com/conan-io/conan/blob/9d48c916ca51713945812bf8b5668ddcd9a303c5/conans/client/graph/provides.py#L16